edge-apps/google-calendar/screenly_qc.yml

```diff
@@ -0,0 +1,91 @@
+---
+syntax: manifest_v1
+description: Google Calendar Edge App QC
+icon: https://playground.srly.io/edge-apps/google-calendar/src/img/icon.svg
+author: Screenly, Inc.
+ready_signal: true
+settings:
+  access_token:
+    type: secret
+    title: Access Token
+    optional: true
+    help_text:
+      properties:
+        help_text: For testing only. In production, the access token should be fetched dynamically via the API.
+        type: oauth:google_calendar:access_token
+      schema_version: 1
+  calendar_id:
+    type: string
+    default_value: primary
+    title: Google Calendar ID
+    optional: true
+    help_text:
+      properties:
+        help_text: Enter the Google Calendar ID to display events from (defaults to primary calendar).
+        type: oauth:google_calendar:calendar_id
+      schema_version: 1
+  calendar_mode:
+    type: string
+    default_value: monthly
+    title: Calendar Mode
+    optional: true
+    help_text: |
+      Select the calendar view mode ('monthly', 'weekly', or 'daily').
+  display_errors:
+    type: string
+    default_value: 'false'
+    title: Display Errors
+    optional: true
+    help_text:
+      properties:
+        advanced: true
+        help_text: For debugging purposes to display errors on the screen.
+        type: boolean
+      schema_version: 1
+  enable_analytics:
+    type: string
+    default_value: 'true'
+    title: Enable Analytics
+    optional: true
+    help_text: Enable or disable Sentry and Google Analytics integrations.
+    is_global: true
+  override_locale:
+    type: string
+    default_value: en
+    title: Override Locale
+    optional: true
+    help_text: |
+      Enter the locale to override the default locale for the calendar.
+      If left blank, the default locale will be used.
+      Example: 'fr'
+  override_timezone:
+    type: string
+    default_value: ''
+    title: Override Timezone
+    optional: true
+    help_text: |
```

Comment:
If there is a special type for timezone selection, let's reuse them. 

@iamplugged , do we have one?